x-pack/plugins/ml/server/models/data_recognizer/modules/metrics_ui_hosts/manifest.json

```diff
@@ -0,0 +1,59 @@
+{
+    "id": "metrics_ui_hosts",
+    "title": "Metrics Hosts",
+    "description": "Detect anomalous memory, cpu, and network behavior on hosts.",
+    "type": "Metricbeat Data",
+    "logoFile": "logo.json",
+    "defaultIndexPattern": "metricbeat-*",
+    "query": {
```

Comment:
If the jobs in this module provide no value without the specific overrides we are expecting from the Metrics UI, then removing this `query` block is the way to hide it from the ML job wizards.

Reply:
the `query` and `defaultIndexPattern` fields have been removed, as we do in the logs integration modules.